shard-operator/src/main/java/com/redhat/service/smartevents/shard/operator/resources/BridgeExecutorSpec.java

```diff
@@ -11,6 +11,8 @@ public class BridgeExecutorSpec {
 
     private String customerId;
 
+    private String processorType;
```

Comment:
@kostola Doesn't this require our CRDs to be regenerated with the additional `processorType` field for the spec?

Reply:
The new CRDs are generated in the `target` folder when you compile the application. When you merge the new code the bot will include the CRDs update in the `update kustomization images` pull request